### ISSUE TITLE
Suggestion: Could you rename Sales in more generic naming?

### ISSUE THREAD

**Issue by biapar** [workcontrolgit/devkit-apiresources-salesapi#2]
Suggestion: Could you rename Sales in more generic naming?

**Comment by workcontrolgit** [comment]
Good idea.  Do you have a suggestion?   I like to make it into a template but don't know how to do it (yet).

**Comment by biapar** [comment]
I think Services, Cloud, BackEnd...
Rename files and projects name and classes with Visual Studio refactory functions.

**Comment by workcontrolgit** [comment]
@biapar - I figured out the template and created one https://github.com/workcontrolgit/TemplateKissAPI.  The zip file KissApi.zip can be downloaded and copied to VS template folder.   Using the template, you can generate the solution and specify the project name.  It will create a solution with and prefix the project name for core, application, infrastructure and webapi accordingly.   Give it a try.  I hope this will save time to start new project.

**Comment by workcontrolgit** [comment]
I created a VS Template and uploaded it to Visual Studio Marketplace.  For instruction to use the template, see [my blog on Medium](https://fuji-nguyen.medium.com/rapid-prototype-asp-net-708df7a59a3?sk=6965ff020c90a8fa40b67254998aaf83)
